docs-src/content/architecture/adrs/adr-0003.md

```diff
@@ -8,7 +8,7 @@ weight = 10003
 sort_by = "weight"
 +++
 
-{{ star() }}
+
```

Comment:
@csoylu can you please revert your changes in markdown files, otherwise we'll badly bump into merge conflicts.

Thanks 🙏 .

I can remove those stars later :) .